airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/config.py

```diff
@@ -42,6 +42,17 @@ class BackwardCompatibilityTestsConfig(BaseConfig):
     )
 
 
+class OAuthTestConfig(BaseConfig):
+    oauth = Field(True, description="Allow source to have another default method that OAuth.")
+    bypass_reason: Optional[str] = Field(description="Reason why OAuth is not default method.")
+
+    @validator("oauth", always=True)
+    def validate_exact_order(cls, oauth, values):
```

Comment:
why is this called `validate_exact_order`? This should be `oauth` or something presumably

Reply:
updated